docs/contributing.md

```diff
@@ -9,10 +9,10 @@ interactions with the project.
 
 ## Code Style
 
-To ensure everybody can work easily and focus in the code, make easy the review process and keep the same style, we have a couple of files defining our code standards:
-- Intellij editor style at `config/checkstyle/OSS_style_idea.xml`
-- Checkstyle configuration files at `checkstyle.xml`
-  - suppression file at `config/checkstyle/OSS_checkstyle/OSS_checkstyle_suppressions.xml`
+To ensure everybody can work easily and focus in the code, make easy the review process and keep the same style. We have a couple of files defining our code standards:
+- IntelliJ editor style at `config/checkstyle/OSS_style_idea.xml`.
+- Checkstyle configuration files at `checkstyle.xml`.
+  - Suppression file at `config/checkstyle/OSS_checkstyle/OSS_checkstyle_suppressions.xml`.
```

Comment:
Maybe is not an error, but I think it should be better to move to move this line to the left so that it is below the others

Reply:
Done.